extra/1-currency-conversion.js

```diff
@@ -15,7 +18,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(pound) {
+  let poundToBrl = pound * 5.643;
+  return poundToBrl.toFixed(2);
+}
```

Comment:
This is good, you're almost there! Just look closely at the requirements again and make sure you're following the instructions 😉